oauthenticator/github.py

```diff
@@ -260,6 +260,15 @@ async def _check_membership_allowed_organizations(
             )
         return False
 
+    def _build_check_membership_url(
+        self, org: str, username: str, team_identifier: str = ":"
```

Comment:
Is there a particular reason for making `team_identifier` a parameter? If there isn't I think it'd be cleaner to hard code it, especially as this is an non-public `_` function.

Reply:
Sounds good to me, will amend!

Reply:
Done.